get-started/kube-deploy.md

```diff
@@ -44,7 +44,7 @@ All containers in Kubernetes are scheduled as _pods_, which are groups of co-loc
         spec:
           containers:
           - name: bb-site
-            image: bulletinboard:1.0
+            image: bulletinboard/bulletinboard:1.0
```

Comment:
This probably isn't correct; this `bulletinboard` image was an image that was built as part of the previous getting started tutorial (https://github.com/docker/docker.github.io/pull/10218), which was replaced with a new "getting started".

We probably need to rewrite this Kubernetes tutorial to use a different image

Reply:
Sounds good! Thank you @thaJeztah